doc/source/cluster/kubernetes/user-guides/kuberay-gcs-ft.ipynb

```diff
@@ -0,0 +1,900 @@
+    "Therefore, KubeRay automatically injects the environment variable `RAY_external_storage_namespace` to all Ray Pods managed by the RayCluster with the RayCluster's UID as the external storage namespace by default.\n",
+    "See [this section](https://docs.ray.io/en/latest/cluster/kubernetes/user-guides/kuberay-gcs-ft.html#kuberay-external-storage-namespace) to learn more about the option.\n",
+    "\n",
+    "### Step 7: Kill the GCS process in the head Pod\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "297712df-81e6-40b8-80e4-cb43fd125dfd",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "remove-cell",
+     "nbval-ignore-output",
+     "remove-output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "export YOUR_WORKER_POD=$(kubectl get pods -l ray.io/group=small-group -o jsonpath='{.items[0].metadata.name}')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "775d8da9-cea7-4f93-adec-653520af37d3",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "nbval-ignore-output",
+     "remove-output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Step 7.1: Check the `RAY_gcs_rpc_server_reconnect_timeout_s` environment variable in both the head Pod and worker Pod.\n",
+    "kubectl get pods $HEAD_POD -o=jsonpath='{.spec.containers[0].env}' | jq\n",
+    "# [Expected result]:\n",
+    "# No `RAY_gcs_rpc_server_reconnect_timeout_s` environment variable is set. Hence, the Ray head uses its default value of `60`.\n",
+    "kubectl get pods $YOUR_WORKER_POD -o=jsonpath='{.spec.containers[0].env}' | jq\n",
+    "# [Expected result]:\n",
+    "# KubeRay injects the `RAY_gcs_rpc_server_reconnect_timeout_s` environment variable with the value `600` to the worker Pod.\n",
+    "# [\n",
+    "#   {\n",
+    "#     \"name\": \"RAY_gcs_rpc_server_reconnect_timeout_s\",\n",
+    "#     \"value\": \"600\"\n",
+    "#   },\n",
+    "#   ...\n",
+    "# ]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "4b56c6fa-fafa-4f4a-8ef0-3513bc49c9f9",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "nbval-ignore-output",
+     "remove-output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Step 7.2: Kill the GCS process in the head Pod.\n",
+    "kubectl exec -i $HEAD_POD -- pkill gcs_server"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "72a46dd5-9305-40c1-b378-456c260c8f36",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "remove-cell",
+     "nbval-ignore-output",
+     "remove-output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "sleep 60\n",
+    "kubectl wait --for=condition=ready pod/$HEAD_POD --timeout=500s\n",
```

Comment:
```suggestion
    "kubectl wait --for=condition=ready pod/$HEAD_POD --timeout=500s\n",
```

Don't need to sleep because we've already use `kubectl wait`.